src/components/views/spaces/SpaceTreeLevel.tsx

```diff
@@ -95,9 +95,9 @@ export const SpaceButton = forwardRef<HTMLElement, IButtonProps>(
         }
 
         let notifBadge;
-        if (notificationState) {
+        if (space && notificationState) {
```

Comment:
Not sure if this interests you, but there's now a `Typeguards.ts` file that has an `isNotUndefined` helper typeguard for situations like this.